cmd/fuzzbinder/main.go

```diff
@@ -51,7 +53,7 @@ func main() {
 	}
 
 	// Generate the fuzzing claim.
-	statement, err := fuzzbinder.GenerateFuzzClaim(fuzzParameters)
+	statement, err := fuzzbinder.GenerateFuzzClaim(fuzzParameters, *validityDays)
```

Comment:
I assume it does not make sense to add the "validity days" to the fuzzParameters?

Reply:
I don't think so. The fuzzParameters are grouped together to be used for the fuzzscraper. The validityDays is used only once.